cluster-up/cluster/k8s-1.17/dev-guide.md

```diff
@@ -25,59 +25,63 @@ It gives us isolation advantage and state freezing of the needed components, all
+* k8s-1.17 dnsmasq container - a container that run dnsmasq, which gives dns and dhcp services.
 
 The containers are running and looks like this:
 ```
-[root@modi01 1.17.0]# docker ps
-CONTAINER ID        IMAGE                   COMMAND                  CREATED             STATUS              PORTS                                                                                                                          NAMES
-3589e85efc7d        kubevirtci/k8s-1.17.0   "/bin/bash -c '/vm.s…"   About an hour ago   Up About an hour                                                                                                                                   k8s-1.17.0-node01
-4742dc02add2        registry:2.7.1          "/entrypoint.sh /etc…"   About an hour ago   Up About an hour                                                                                                                                   k8s-1.17.0-registry
-13787e7d4ac9        kubevirtci/k8s-1.17.0   "/bin/bash -c /dnsma…"   About an hour ago   Up About an hour    127.0.0.1:8443->8443/tcp, 0.0.0.0:32794->2201/tcp, 0.0.0.0:32793->5000/tcp, 0.0.0.0:32792->5901/tcp, 0.0.0.0:32791->6443/tcp   k8s-1.17.0-dnsmasq
+[root@modi01 1.17]# docker ps
+CONTAINER ID   IMAGE                                            COMMAND                  CREATED              STATUS              PORTS                                                                                                                                                                         NAMES
+8ddefc88cdd2   quay.io/kubevirtci/k8s-1.17:2103240101-142f745   "/bin/bash -c '/vm.s…"   About a minute ago   Up About a minute                                                                                                                                                                                 k8s-1.17-node01
+1e10735ba935   registry:2.7.1                                   "/entrypoint.sh /etc…"   About a minute ago   Up About a minute                                                                                                                                                                                 k8s-1.17-registry
+930002ada03f   quay.io/kubevirtci/k8s-1.17:2103240101-142f745   "/bin/bash -c /dnsma…"   About a minute ago   Up About a minute   127.0.0.1:8443->8443/tcp, 0.0.0.0:49189->80/tcp, 0.0.0.0:49188->443/tcp, 0.0.0.0:49187->2201/tcp, 0.0.0.0:49186->5000/tcp, 0.0.0.0:49185->5901/tcp, 0.0.0.0:49184->6443/tcp   k8s-1.17-dnsmasq
 ```
 
 Nodes:
 ```
 [root@modi01 kubevirtci]# oc get nodes
-NAME     STATUS   ROLES    AGE   VERSION
-node01   Ready    master   83m   v1.17.0
+NAME     STATUS   ROLES           AGE   VERSION
+node01   Ready    master,worker   58s   v1.17.16-rc.0
 ```
 
 # Inner look of a deployed cluster
 We can connect to the node of the cluster by:
 ```
+[ -z "$KUBEVIRTCI_TAG" ] && export KUBEVIRTCI_TAG=$(curl -L -Ss https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirtci/latest)
```

Comment:
is this only on k8s-1.17? 
on my  k8s 1.20  I was able to connect to the node without exporting `KUBEVIRTCI_TAG`?

```
[root@zeus09 kubevirt]# env | grep KUBEVIRTCI_TAG
[root@zeus09 kubevirt]# cluster-up/ssh.sh node01
2021/04/07 15:39:54 Waiting for host: 192.168.66.101:22
2021/04/07 15:39:54 Connected to tcp://192.168.66.101:22
Last login: Tue Apr  6 16:28:00 2021 from 192.168.66.2
```

Reply:
You are on kubevirt, which exports it in its own scripts, please check kubevirtci (this PR is of kubevirtci repo)

```
kubevirtci (master) $ ./cluster-up/ssh.sh node01
FATAL: either KUBEVIRTCI_TAG or KUBEVIRTCI_GOCLI_CONTAINER must be set
kubevirtci (master) $ 
kubevirtci (master) $ 
kubevirtci (master) $ nodes
NAME     STATUS   ROLES                  AGE    VERSION
node01   Ready    control-plane,master   110s   v1.20.5
node02   Ready    worker                 47s    v1.20.5
kubevirtci (master) $ 
```

Reply:
got it. Thanks.